Doxyfile

```diff
@@ -0,0 +1,38 @@
+#---------------------------------------------------------------------------
+# Project related configuration options
+#---------------------------------------------------------------------------
+PROJECT_NAME           = "dcurl"
+PROJECT_NUMBER         = 0.1.0
+OUTPUT_DIRECTORY       = docs
+OPTIMIZE_OUTPUT_FOR_C  = YES
+#---------------------------------------------------------------------------
+# Configuration options related to warning and progress messages
+#---------------------------------------------------------------------------
+QUIET                  = YES
+#---------------------------------------------------------------------------
+# Configuration options related to the input files
+#---------------------------------------------------------------------------
+INPUT                  = . \
+                         docs \
+                         src
+FILE_PATTERNS          = dcurl.c \
+                         dcurl.h \
```

Comment:
I think documentation could just all put in dcurl.h

Reply:
I searched on the Internet and found out that writing the document in C source file or header file is acceptable.
It only suggests that we should not write the document on both side.

The reason I wrote in source file is if the functions are changed, they can be modified without opening another file.

I think no matter which way we choose, just make sure every project follows the same rule.

Reply:
The comment has been moved to the header file.

Reply:
I did not remove the `dcurl.c` in `FILE_PATTERNS` since I want to make the users to link and check the source code easily.